tests/util/kubectl/kubectl.go

```diff
@@ -351,6 +351,43 @@ func DumpLogs(path string, namespace string) KCmd {
 	return KCmd{Command: "cluster-info", Args: args, Flags: flags}
 }
 
+// DumpClusterInfo Executes `kubectl cluster-info dump -o yaml` on each cluster. The output
+// is stored under <project-root>/build/test.
+func DumpClusterInfo(path string, namespace string) error {
```

Comment:
It would be nice to put the output under a test-specific subdirectory. I realize though that it would require passing the test name through several functions in the ginkgo package. If there is a desire to do that it could be done in a later PR.

Reply:
It is done already (the path is test specific with date and test name)